src/Service/Models/PageFilters.php

```diff
@@ -4,11 +4,8 @@
 
 class PageFilters
 {
-    public array $filters = [];
-
-    public function __construct(array $hierarchies)
+    public function __construct(public array $filters)
```

Comment:
```suggestion
    public function __construct(public array $filters = [])
```
Havde også en default før